test_app/lib/screens/join_screen.dart

```diff
@@ -154,13 +157,17 @@ class _JoinConferenceContentState extends State<JoinConferenceContent> {
                     blackText: "External ID  ", colorText: widget.externalId),
                 const SizedBox(height: 16),
                 Form(
-                  key: formKeyAlias,
-                  autovalidateMode: AutovalidateMode.onUserInteraction,
-                  child: InputTextFormField(
-                      labelText: 'Conference alias',
-                      controller: conferenceAliasTextController,
-                      focusColor: Colors.deepPurple),
-                ),
+                    key: formKeyAlias,
+                    autovalidateMode: AutovalidateMode.onUserInteraction,
+                    child: InputTextFormField(
+                        labelText: 'Conference alias',
+                        controller: conferenceAliasTextController,
+                        focusColor: Colors.deepPurple,
+                        isStorageNeeded: true,
+                        onStorageIconTap: () async {
+                          await showAliasSelectorDialog(context,
+                              SharedPreferencesHelper().conferenceAliases);
+                        })),
```

Comment:
Let's use trailing commas for proper formatting.

```suggestion
                    key: formKeyAlias,
                  autovalidateMode: AutovalidateMode.onUserInteraction,
                  child: InputTextFormField(
                    labelText: 'Conference alias',
                    controller: conferenceAliasTextController,
                    focusColor: Colors.deepPurple,
                    isStorageNeeded: true,
                    onStorageIconTap: () async {
                      await showAliasSelectorDialog(
                        context,
                        SharedPreferencesHelper().conferenceAliases,
                      );
                    },
                  ),
                ),
```